src/lib/modernExtend.ts

```diff
@@ -1052,12 +1052,25 @@ export function iasZoneAlarm(args: IasArgs): ModernExtend {
             };
 
             if (bothAlarms) {
-                payload = {[alarm1Name]: (zoneStatus & 1) > 0, ...payload};
-                payload = {[alarm2Name]: (zoneStatus & 1 << 1) > 0, ...payload};
+                if (args.zoneType === 'contact') {
```

Comment:
This line is duplicated 3 times, I suggest to add a variable below `const exposeList`: `const inverseAlarm = args.zoneType === 'contact'`